### ISSUE TITLE
Request to track the “5-second-rule” for mana-regeneration

### ISSUE THREAD

**Issue by Leithlot** [kkthnx-wow/KkthnxUI_Classic#55]
Hi,

I would like to suggest/request a feature to be implemented with the mana ticks setting. If possible, rather than the spark being frozen for 5 seconds after spending mana it can be tracked? Either numerically or by using the spark.

For reference:
[https://wow.gamepedia.com/Five_second_rule](url)
[https://www.curseforge.com/wow/addons/fivesecondrule](url)

Cheers

**Comment by Kkthnx** [comment]
This is already in the UI

**Comment by Leithlot** [comment]
Thanks for the quick response, 

I just looked through all the setting and I found nothing pertaining to the five second rule. I assume it must not be working correctly with my UI. Currently, after I cast a spell the spark just freezes, then after 5 seconds, resets to the left side of the mana bar to then tick mana regeneration. My original suggestion was to show some indicator of this 5 second window. How do I implement this in your UI?
